bundles/org.codechecker.eclipse.plugin/src/org/codechecker/eclipse/plugin/config/project/CodeCheckerProject.java

```diff
@@ -303,4 +409,48 @@ public void configUpdated(Map<ConfigTypes, String> config) {
     public boolean isGlobal() {
         return isGlobal;
     }
+
+    /**
+     * Pojo for compilation command records. For the time being one
+     * {@link ComilationCommand} equals the other if their file members equals.
+     */
+    private static class ComilationCommand {
+        String directory;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/CodeChecker/CodeCheckerEclipsePlugin/pullRequest?prid=4641621)